admin/src/Components/SelectInput.jsx

```
@@ -1,80 +1,77 @@
-            />
-        );
+    const select = (
+        <Select
+            id={props.name}
+            name={props.name}
+            className="uk-width-1-1"
+            options={options}
+            value={options.find((o) => o.id === value) || null}
+            getOptionValue={props.getValue}
+            getOptionLabel={props.getLabel}
+            onChange={(o) => (props.model[props.name] = props.getValue(o))}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)